getting-started/data-node-authentication.md

```diff
@@ -0,0 +1,70 @@
+# Data Node Authentication
+
+In a multi-node environment, it is necessary to ensure that the access
+node have the necessary roles and permissions on all the data nodes
+where distributed hypertables are placed.
+
+This is critical since when TimescaleDB is executing a query or
+command it will use the same role on the data nodes for the
+fragments. This means that the same role need to exist on the data
+nodes and authenticate to them.
+
+In a typical setup, you will deploy the distributed hypertable in a
+database and use the same database on all the data nodes. This means
+that we need to ensure that a user that is connected to the access
+node can access the same databases on the data nodes.
+
+## Setting up password authentication
+
+In order to set up password authentication for a user `new_user`, it
+is necessary to update the HBA (Host-Based Authentication) file,
+typically named `pg_hba.conf`, on the data nodes to enable password
+authentication for `new_user`. Details of the format of `pg_hba.conf`
+can be found in the [PostgreSQL manual, Chapter 20.1][postgresql-hba].
+
+In order to give `new_user` access to all databases on the data node,
+add the following line to the `pg_hba.conf` on the data node. This
+will use SCRAM SHA-256 password authentication. For other password
+authentication methods, see [PostgreSQL manual, Chapter
+20.5][auth-password].
```

Comment:
I would skip the chapter number reference since this is self-evident by the link.